Backend/templates/gitlab-ci/master-gitlab-ci.yml

```diff
@@ -217,6 +255,8 @@ DocumentationProd:
 
 DeployDocumentationProd:
   extends: .deploy_documentation_template
+  environment:
+    name: production
   variables:
     PROJECT: $PROJECT_PROD
     DOC: "prod"
```

Comment:
Should be upcase?

Reply:
this will be used to format the documentation path 

`documentation gs://$BUCKET_DOCUMENTION/$DOC/$NAME/$BUILD_NUMBER`